openslides/agenda/static/js/agenda/site.js

```diff
@@ -24,12 +24,18 @@ angular.module('OpenSlidesApp.agenda.site', [
 
 .config([
     '$stateProvider',
-    function($stateProvider) {
+    'gettext',
+    function($stateProvider, gettext) {
+        gettext('Agenda');
```

Comment:
Sorry. My patch was just to show that gettext can be injected into the config function. But please set the translation mark to the string in title. We do not want to repeat ourselves here. If gettext is not possible in the object, use a variable.